packages/relay/src/lib/relay.ts

```diff
@@ -124,7 +124,11 @@ export class RelayImpl implements Relay {
 
     client.setTransportSecurity(process.env.CLIENT_TRANSPORT_SECURITY === 'true' || false);
 
-    logger.info(`SDK client successfully configured to ${JSON.stringify(hederaNetwork)} for account ${client.operatorAccountId}`);
+    const sdkRequestTimeout = parseInt(process.env.SDK_REQUEST_TIMEOUT || '');
+    client.setRequestTimeout(Number.isInteger(sdkRequestTimeout) ? sdkRequestTimeout : 1000);
```

Comment:
Is this a simpler approach to then skip the `Number.isInteger`?
```suggestion
    client.setRequestTimeout(parseInt(process.env.SDK_REQUEST_TIMEOUT || '1000'));
```

Reply:
Updated.